src/components/utils/BackButton.js

```diff
@@ -13,8 +13,7 @@ import { Button, IconButton, useMediaQuery, useTheme } from "@material-ui/core";
 
 import { ArrowBack } from "@material-ui/icons";
 
-export default function BackButton(props) {
-    const { dirty = false } = props;
+export default function BackButton({ dirty = false, ...props }) {
```

Comment:
Since `props` are spread down into the inner `Button` (which ignore this `dirty` prop and passes it on to the DOM), there was a DOM warning about the `dirty` prop before this change.